index.html

```diff
@@ -10,6 +10,7 @@
 </head>
 
 <body>
+
     <div class="container">
         <div class="col-md-12">
```

Comment:
<h3>Optional</h3>

Good on using div tag to hold your items, but I🤔 suggest making your project more semantic by tags like header, nav, section, and footer, You used div tag instead of section tag.